### DISCUSSION TITLE
Is it possible to set the postgres schema?

### DISCUSSION THREAD

**Discussion by gee-shibasaki** [collerek/ormar#651] [Q&A]
I'm using postgres which is not defualt public schema.

Is it possible to set the schema?

like this.
```
class User(ormar.Model):
    class Meta:
        schemaname: str = "myschema" ← set schema
        tablename: str = "users"
        metadata = metadata
        database = database

    id: int = ormar.Integer(primary_key=True)
    email: str = ormar.String(max_length=255, nullable=False)
```

If currently not possible. Could you tell me a better way to achieve this?

I want to insert data into the table in a schema other than the public which is default.

**Comment by SepehrBazyar** [comment]
Hi @gee-shibasaki 
I did not understand what you want to do and what you need
please explain more.

**Reply by hk3dva** [reply]
If I understood correctly. The author wants to use posgresql's ability to split data into several schemas. A schema is an area in the database that divides tables among themselves, for example, a schema with Users, News, Orders. Schemas can contain their own tables and serve only to separate domains from each other.
I would like to know whether working with circuits is implemented in ormar or not
